packages/expo-image/ios/ImageModule.swift

```diff
@@ -39,7 +39,7 @@ public final class ImageModule: Module {
       }
 
       Prop("contentFit") { (view, contentFit: ContentFit?) in
-        view.contentFit = contentFit ?? .cover
+        view.contentFit = contentFit!
```

Comment:
#### 🔴 SwiftLint: [Force Unwrapping Violation](https://realm.github.io/SwiftLint/force_unwrapping.html)
Force unwrapping should be avoided